integrations/cohere/src/haystack_integrations/components/embedders/cohere/text_embedder.py

```diff
@@ -99,7 +109,6 @@ def to_dict(self) -> Dict[str, Any]:
             input_type=self.input_type,
             api_base_url=self.api_base_url,
             truncate=self.truncate,
-            use_async_client=self.use_async_client,
```

Comment:
I have the same comment here about backwards compatibility as in the document embedder.

Reply:
done